x-pack/plugin/sql/qa/server/src/main/resources/filter.sql-spec

```diff
@@ -129,3 +129,11 @@ whereWithInAndNullHandling1
 SELECT last_name l FROM "test_emp" WHERE languages in (2, 10)AND (emp_no = 10018 OR emp_no = 10019 OR emp_no = 10020) ORDER BY emp_no;
 whereWithInAndNullHandling2
 SELECT last_name l FROM "test_emp" WHERE languages in (2, null, 10) AND (emp_no = 10018 OR emp_no = 10019 OR emp_no = 10020) ORDER BY emp_no;
+
+whereWithInAndOneDatetimeValue
+SELECT birth_date FROM "test_emp" WHERE birth_date IN ('1959-12-25T12:12:12.000Z'::datetime);
+whereWithInAndMultipleDatetimeValues
+SELECT birth_date FROM "test_emp" WHERE birth_date IN ('1959-12-25T12:12:12.000Z'::datetime, '1959-07-23T12:12:12.000Z'::datetime);
+
+whereWithInAndTimeValue
+SELECT birth_date FROM "test_emp" WHERE birth_date::time IN ('12:12:12.000'::time);
```

Comment:
This new addition is good, but if you actually the run the tests, it will fail due to #63521, which will need fixing before such a test can be merged.